controllers/ncControllers.js

```diff
@@ -0,0 +1,23 @@
+const fs = require("fs");
+const { selectTopics } = require("../models/ncModel.js");
+
+exports.getTopics = (request, response, next) => {
+  selectTopics()
+    .then((topics) => {
+      response.status(200).send({ topics: topics });
+    })
+    .catch((error) => {
+      next(error);
+    });
+};
+
+exports.fetchEndpoints = (request, response, next) => {
+  return fs.readFile("endpoints.json", "utf-8", (error, data) => {
```

Comment:
Here you are still reading the file but you need to be making a get request to the api and then comparing the files in your test.